test/WebJobs.Script.Tests/Helpers/SimpleWebTokenTests.cs

```diff
@@ -89,6 +90,24 @@ public void Validate_Token_Uses_Website_Encryption_Key_If_Container_Encryption_K
             Assert.True(SimpleWebTokenHelper.TryValidateToken(token, new SystemClock()));
         }
 
+        [Fact]
+        public void Validate_Token_Checks_Signature_If_Signature_Is_Available()
```

Comment:
Rather than a Token based test here, I'd rather see an Encrypt/Decrypt test, since we're not really dealing with SWTs in your scenario.

Reply:
Added the test for encrypting and decrypting a mock host assignment context